avalon_listener/listener/tcs_listener.py

```diff
@@ -259,36 +261,49 @@ def parse_command_line(config, args):
         config['Logging']['LogLevel'] = options.loglevel.upper()
     if options.bind_uri:
         bind_uri = options.bind_uri
+    else:
+        bind_uri = config["Listener"]["listener_port"]
     if options.lmdb_url:
-        config["KvStorage"]["remote_url"] = options.lmdb_url
+        config["KvStorage"]["remote_storage_url"] = options.lmdb_url
 
     return bind_uri
 
 # -----------------------------------------------------------------
 # -----------------------------------------------------------------
 
 
+def get_config_dir():
+    """
+    Returns the avalon configuration directory based on the
+    TCF_HOME environment variable (if set) or OS defaults.
+    """
+    if 'TCF_HOME' in os.environ:
+        return os.path.join(os.environ['TCF_HOME'], 'avalon_listener/')
+
+    return '/etc/avalon'
+
+
 def main(args=None):
     import config.config as pconfig
     import utility.logger as plogger
 
     # parse out the configuration file first
-    conffiles = ['tcs_config.toml']
-    confpaths = [".", TCFHOME + "/config"]
+    conffile = ['listener_config.toml']
+    confpath = [get_config_dir()]
```

Comment:
conffile -> conf_file
confpath -> conf_path

Reply:
This is in line with recently merged PR (https://github.com/hyperledger/avalon/pull/194), File: examples/shared_kv_storage/kv_storage/remote_lmdb/lmdb_listener.py, Line no: 131, 132

Reply:
the review of this variable name got missed that PR. Avalon follows PEP8 style for variable names. https://www.python.org/dev/peps/pep-0008/#naming-conventions